linkerd/app/src/env.rs

```diff
@@ -998,6 +1063,37 @@ mod tests {
         assert_eq!(parse_duration("1"), Err(ParseError::NotADuration));
     }
 
+    #[test]
+    fn convert_labels_to_map_different_values() {
+        let labels_string = "cluster=\"test-cluster1\"\nrack=\"rack-22\nzone=us-est-coast\nlinkerd.io/control-plane-component=\"controller\"\nlinkerd.io/proxy-deployment=\"linkerd-controller\"\nworkload=\nkind=\"\"\nkey1=\"=\"\nkey2==value2\nkey3\n=key4\n
```

Comment:
This would be a bit more readable as a multiline string: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=daa8a3650244073182fc93cde9c7a77d

```suggestion
        let labels_string =  "\
cluster=\"test-cluster1\"
rack=\"rack-22
zone=us-est-coast
linkerd.io/control-plane-component=\"controller\"
linkerd.io/proxy-deployment=\"linkerd-controller\"
workload=
kind=\"\"
key1=\"=\"
key2==value2
key3
=key4
";
```

Reply:
Alternatively, we could use the line continuation `\` syntax and also indent each line:

```suggestion
        let labels_string =  "\
            cluster=\"test-cluster1\"\n\
            rack=\"rack-22\"\n\
            zone=us-est-coast\n\
            linkerd.io/control-plane-component=\"controller\"\n\
            linkerd.io/proxy-deployment=\"linkerd-controller\"\n\
            workload=\n\
            kind=\"\"\n\
            key1=\"=\"\n\
            key2==value2\n\
            key3\n\
            =key4\n\
            ";
```
Although I don't know if that's significantly clearer.